addons/hr_contract/models/hr_contract.py

```diff
@@ -127,7 +128,12 @@ def _check_current_contract(self):
 
             domain = expression.AND([domain, start_domain, end_domain])
             if self.search_count(domain):
-                raise ValidationError(_('An employee can only have one contract at the same time. (Excluding Draft and Cancelled contracts)'))
+                raise ValidationError(
+                    _(
+                        'An employee can only have one contract at the same time. (Excluding Draft and Cancelled contracts).\n\nEmployee: %(employee_name)s',
```

Comment:
This requires `.pot` modification.

Reply:
I mean, better to not include this part...

Reply:
@MiquelRForgeFlow is right, no need to break the translations for that :)

Please only keep the domain part. 

Thanks for the fix @etobella

Reply:
Done!